msg/LineIntersectionInImage.msg

```diff
@@ -8,6 +8,9 @@ uint8 X=3
 
 uint8 type
 
```

Comment:
Why do messages like these `...InImage` or `Relative` have no timestamp of the original image?

Reply:
because they are mostly parts of some other message like LineInformationInImage or something.